Chinese/大话西游.md

```diff
@@ -157,3 +157,4 @@
 [你给我站住](stop/stop.md)
 
 [Gao Shen Me](gaoshenme/gaoshenme.md)
+[Js不可告人的秘密](Javascript/js.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.